eng/pipelines/templates/steps/vcpkg.yml

```diff
@@ -1,146 +1,58 @@
-# Assumes that the VM image is in a variable called OSVmImage
-
 parameters:
   # Use the variable name itself (no $() or other wrapping syntax). This is
   # because we use runtime and macro expressions for conditions and script
   # invocations
   DependenciesVariableName: vcpkg.deps
 
-  # Ref at which to check out (can be commit SHA or tag in the form of
-  # `tags/<tag>`)
-  VcpkgVersion: tags/2020.06
-
 steps:
-  # Set VCPKG_INSTALLATION_ROOT location for cache on Mac OS hosts
-  - task: Bash@3
-    inputs:
-      targetType: inline
-      workingDirectory: $(Agent.TempDirectory)
-      script: |
-        echo "##vso[task.prependpath]$(Agent.TempDirectory)/vcpkg"
-        echo "##vso[task.setvariable variable=VCPKG_INSTALLATION_ROOT;]$(Agent.TempDirectory)/vcpkg"
-    displayName: MacOS - Set VCPKG_INSTALLATION_ROOT
+  - pwsh: |
+      $TargetPath = "$(Agent.TempDirectory)/vcpkg"
+      Remove-Item -Path $TargetPath -Recurse -Force -ErrorAction Ignore
```

Comment:
Is it safe to ignore the error and continue if we can't remove that directory? Should we fail the build at this point?